api/configserver/save.go

```diff
@@ -145,6 +168,71 @@ func (s *Server) SaveConfig(w http.ResponseWriter, r *http.Request) {
 	s.writeSaveConfigResponse(w, SaveConfigResponse{Warnings: warnings}, session)
 }
 
+// Simply validate that the credentials exist; don't do anything with the actual secrets
+func validateCredParams(vars creds.Variables, config atc.Config, session lager.Logger) []string {
+	// 1. currently doesn't support tasks specified using a task file
+	// 2. should we check each of these are not nil first before Evaluating them to prevent unnecessary lookups, or is this already cached in vars?
+
+	errs := []string{}
+
+	for _, resourceType := range config.ResourceTypes {
+		_, err := creds.NewSource(vars, resourceType.Source).Evaluate()
+		if err != nil {
+			session.Debug(err.Error())
```

Comment:
Let's clean up these debug lines - can we just have one log line at the end with the aggregate of the errors?:

Something like:

```go
if err != nil { // assuming use of mutlierror.Append
  session.Info("config-has-invalid-creds", lager.Data{"errors": err.Error()})
}
```